code/datums/achievements/misc_achievements.dm

```diff
@@ -75,3 +83,15 @@
 	name = "One Lean, Mean, Cleaning Machine"
```

Comment:
question: why are some of these achievement names capitalized, but not others?

Reply:
because someone else added them